Xamarin.Forms.Platform.UAP/Forms.cs

```diff
@@ -40,18 +40,22 @@ public static void Init(IActivatedEventArgs launchActivatedEventArgs, IEnumerabl
 			Device.SetFlags(s_flags);
 			Device.Info = new WindowsDeviceInfo();
 
-			switch (DetectPlatform())
+			switch (Windows.System.Profile.AnalyticsInfo.VersionInfo.DeviceFamily)
 			{
-				case Windows.Foundation.Metadata.Platform.Windows:
+				case "Windows.Desktop":
 					Device.SetIdiom(TargetIdiom.Desktop);
 					break;
-				case Windows.Foundation.Metadata.Platform.WindowsPhone:
+				case "Windows.Mobile":
 					Device.SetIdiom(TargetIdiom.Phone);
 					break;
+				case "Windows.Xbox":
+					Device.SetIdiom(TargetIdiom.TV);
+					break;
 				default:
 					Device.SetIdiom(TargetIdiom.Tablet);
```

Comment:
I'm not sure that the tablet is the correct device by default. The number of device types (`Windows.Holographic`, `Windows.Team`, `Windows.IoT`) may increase and not all of them will be compatible with the tablet.

What you think @samhouts ? Perhaps set these types of devices as `TargetIdiom.Unsupported`?

Reply:
Hmm. Yeah, I think that sounds like a decent idea.